### ISSUE TITLE
AKRFC Gridded Data Acquistion for Alaska District

### ISSUE THREAD

**Issue by FHanbali** [USACE/cumulus#70]
Pursue LDM data acquisition from AKRFC, including samples like:

- precip_acr_grid_18_00_20210311.1493944413.grb 
- qpf06f_has_102f_20210316_18_awips_202103120755.293358096.grb
- ta01f_has_100f_20210316_16_awips_202103120603.2220943037.grb

**Comment by jeffsuperglide** [comment]
Waiting on Central/Southern Region HQ to allow connections.

**Comment by jeffsuperglide** [comment]
Merged into issue USACE/cumulus#129 and closes #70
